dashboard/src/shared/Config.ts

```diff
@@ -9,6 +9,14 @@ export default class Config {
   public static async getConfig() {
     const url = Config.APIEndpoint;
     const { data } = await axios.get<IConfig>(url);
+
+    // Development environment config overrides
+    // TODO(miguel) Rename env variable to TELEPRESENCE_CONTAINER_NAMESPACE
+    // and remove package.json yarn run mapping once create-react-app is ejected
+    if (process.env.REACT_APP_KUBEAPPS_NS) {
```

Comment:
add `process.env.NODE_ENV === "development"/"test"` check to ensure we never do this in production